exploit_generation_example/crash_analysis.py

```diff
@@ -0,0 +1,96 @@
+# 
+# Usage: python crash_analysis.py ./prog  -- argv_conc -- arg_symbolic
+# 
+# manticore uses '+' to know which parts of the argv are symbolic.
+# In our example it gives:
+# python crash_analysis.py ./bof  -- AAAAAAAAAAAAAAAAAAAAAAA -- +++++++++++++++++++++++ 
+#
+import sys
+from manticore import Manticore
+
+def concrete_run(prog, params):
+    print "Starting concrete execution"
+
+    m = Manticore(prog, params)
+    # 'trace' will contain the executed instructions
+    m.context['trace'] = []
+
+    # None: The hook will be applied to all the instructions
+    @m.hook(None)
+    def record_trace(state):
+        pc = state.cpu.PC
+
+        # Store the instruction
+        m.context['trace'] += [pc]      
```

Comment:
You'll get a warning here as you are using a shared context without a lock. I think it's okay in this case